.eslintrc.js

```diff
@@ -6,23 +6,18 @@ module.exports = {
     node: true,
   },
   extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:import/errors',
+    '@chanzuckerberg/eslint-config-edu-js',
+    '@chanzuckerberg/eslint-config-edu-ts',
     'plugin:import/warnings',
-    'plugin:import/typescript',
     'plugin:prettier/recommended',
     'plugin:testing-library/react',
   ],
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    ecmaVersion: 12,
-    sourceType: 'module',
-  },
   ignorePatterns: ['storybook-static/', 'build/'],
-  plugins: ['@typescript-eslint', 'testing-library', 'import', 'jest'],
+  plugins: ['testing-library', 'jest'],
   rules: {
-    '@typescript-eslint/no-empty-function': 'off',
-    'import/order': ['error', { alphabetize: { order: 'asc' } }],
+    // This lint rule is extremely useful, but does not seem to work in this repo... not sure why 😬
+    // Is it because of the monorepo structure, and eslint can't tell that dev deps at the top level
+    // should be importable by packages?
+    'import/no-extraneous-dependencies': 'off',
```

Comment:
All the other rules seem to work. This wasn't turned on before using the shared config, and this repo doesn't see a ton of development, so I think it's okay to have it off for now.

One day I'd like to get to the bottom of it, but don't want to put the time into it now.